metron-deployment/ansible/roles/ambari_common/tasks/ambari-repo-centos.yml

```diff
@@ -18,5 +18,13 @@
 - name: Setup Ambari repo on CentOS
   get_url: url="{{ centos_ambari_install_url }}" dest=/etc/yum.repos.d/ambari.repo
 
+- name: Repo Cache Settings
+  lineinfile:
+    path: "/etc/yum.repos.d/{{ item }}"
+    regexp: keepcache=.*
+    line: "keepcache=1"
+  with_fileglob:
+    - "/etc/yum.repos.d/*.repo"
```

Comment:
What is this bit for? 

When I look at the current CentOS image from master, I don't see any repo definitions where the `keepcache` parameter is defined.  Since it is not defined, I would not expect this `lineinfile` to do anything.

```
[vagrant@node1 yum.repos.d]$ grep keepcache /etc/yum.repos.d/*.repo
[vagrant@node1 yum.repos.d]$
```

And the default setting of keepcache is already 1, AFAIK.